debs/SPECS/wazuh-agent/debian/postrm

```diff
@@ -51,6 +51,37 @@ case "$1" in
     ;;
 
     upgrade)
+        # If the upgrade downgrades to earlier versions, restore ownership
+        if command -v ${DIR}/bin/ossec-control > /dev/null 2>&1; then
+
+            OSMYSHELL="/sbin/nologin"
+
+            if [ -f ${DIR}/queue/sockets/.agent_info ]; then
+                mv ${DIR}/queue/sockets/.agent_info ${DIR}/queue/ossec/
```

Comment:
Would also be necessary to remove the `sockets` folder?

Reply:
I've updated the script to also delete the `sockets` folder.